ocaml/xapi-cli-server/record_util.ml

```diff
@@ -551,3 +551,11 @@ let vm_placement_policy_of_string a =
       `anti_affinity
   | s ->
       record_failure "Invalid VM placement policy, got %s" s
+
+let repo_origin_to_string = function
```

Comment:
I don't think we need this and we can rely on the auto-generated function now. See the comment on top of this file.